src/test/resources/solr/collection1/conf/schema.xml

```diff
@@ -17,5 +17,6 @@
 
 	<field name="title"     type="string" indexed="true"  stored="true" />
 	<field name="access"    type="string" indexed="false" stored="true" default="c:0" docValues="true" />
+	<field name="accesses"  type="string" indexed="false" stored="true" default="c:0" docValues="true" multiValued="true" />
```

Comment:
Hmm, not sure whether that's needed since you deal with both in the code already. I only see it needed for the tests.

Alternatively we could also just bump the major version to make this a required change.